packages/config/src/bridges/near.ts

```diff
@@ -117,6 +117,16 @@ export const near: Bridge = {
         name: 'ERC20Locker',
         description: 'Escrow contract for ERC20 tokens.',
       },
+      {
+        address: '0x6BFaD42cFC4EfC96f529D786D643Ff4A8B89FA52',
+        name: 'EthCustodian',
+        description: 'Escrow contract for ETH tokens.',
+      },
+      {
+        address: '0x85F17Cf997934a597031b2E18a9aB6ebD4B9f6a4',
+        name: 'NEAR token',
```

Comment:
This escrow does not lock NEAR tokens although (it's a burn-mint type of bridge) so it does not have to be added to the escrow list and NEAR does not contribute to TVL (Tokens Locked in a Bridge).